src/test/java/com/epam/tc/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,156 @@
+package com.epam.tc.hw2.ex1;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class Exercise1 {
+    private WebDriver webDriver;
+
+    private WebElement webElement;
+
+    @BeforeTest
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @AfterTest
+    public void clear() {
+        webDriver.quit();
+    }
+
+    @Test
+    public void firstExerciseTest() throws InterruptedException {
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
```

Comment:
This code should be part of `@Before...` method

Reply:
done